samples/bluetooth/peripheral_status/boards/thingy53_nrf5340_cpuapp_ns.conf

```diff
@@ -55,7 +55,6 @@ CONFIG_HW_STACK_PROTECTION=y
 # Bootloader configuration
 
 CONFIG_BOOTLOADER_MCUBOOT=y
-CONFIG_MCUBOOT_IMAGE_VERSION="2.4.0+0"
```

Comment:
It will actually retain the previous behaviour but needs https://github.com/zephyrproject-rtos/zephyr/pull/61852 brought in, which I will do after the upmerge

Reply:
Just try to get it before the NCS 2.5. Otherwise the release note here may actually be needed.